cli/common.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# usage: increase_minor_version 0.1.1 => 0.2.0
```

Comment:
missing error handling in here (e.g.  0.1 or v0.1.2 )

(actually I saw you were using Python script for the version stuff before, it should be easier to handle that)

Reply:
yes, now it's still using an inline python script. let me fix it.